taipy/gui/gui.py

```diff
@@ -2880,7 +2880,11 @@ def stop(self):
             _TaipyLogger._get_logger().info("Gui server has been stopped.")
 
     def _get_authorization(self, client_id: t.Optional[str] = None, system: t.Optional[bool] = False):
-        return contextlib.nullcontext()
+        try:
+            return _Hooks()._get_authorization(self, client_id, system) or contextlib.nullcontext()
+        except Exception as e:
+            _warn("No Hook:", e)
```

Comment:
I don't quite understand this exception.
Whether there are no hooks, then nullcontext is fine.
Or there are, then live and let live...